test/integration/src/test/java/io/pravega/test/integration/ByteStreamTest.java

```diff
@@ -280,7 +281,47 @@ public void testBlockingRead() throws IOException {
         writer.closeAndSeal();
         assertEquals(-1, reader.read());
     }
-    
+
+    @Test(timeout = 30000)
+    public void testRecreateStream() {
+        String scope = "ByteStreamTest";
+        String stream = "stream";
+
+        StreamConfiguration config = StreamConfiguration.builder().build();
+        @Cleanup
+        StreamManager streamManager = new StreamManagerImpl(controller, null);
+        // create a scope
+        assertTrue("Create scope failed", streamManager.createScope(scope));
+        // create a stream
+        assertTrue("Create stream failed", streamManager.createStream(scope, stream, config));
+        // verify read and write.
+        verify_byte_client_read_write(scope, stream);
+        // delete the stream and recreate
+        assertTrue("Seal stream operation failed", streamManager.sealStream(scope, stream));
+        assertTrue("Delete Stream operation failed", streamManager.deleteStream(scope, stream));
+        assertTrue("Recreate stream failed", streamManager.createStream(scope, stream, config));
+        // verify read and write.
+        verify_byte_client_read_write(scope, stream);
+    }
+
+    private void verify_byte_client_read_write(String scope, String stream) {
```

Comment:
please use Java conventions for method names:
verifyByteClientReadWrite

Reply:
fixed it.